lib/config/ui_config.dart

```diff
@@ -203,3 +203,11 @@ abstract class NavigationTabViewConfig {
   static const radius = 8.0;
   static const navIconSize = 30.0;
 }
+
+abstract class BuildingDetailConfig {
```

Comment:
I'd change naming to `horizontalPaddingLarge ``borderRadiusSmall `etc.. The goal here's not to include separate components name because paddings, heights and so on, could be used somewhere else at the screen.